torchtext/experimental/vectors.py

```diff
@@ -58,22 +60,29 @@ def fast_text(language="en", unk_tensor=None, root='.data'):
 
     """
     url = 'https://dl.fbaipublicfiles.com/fasttext/vectors-wiki/wiki.{}.vec'.format(language)
-    vectors_file_path = os.path.join(root, os.path.splitext(os.path.basename(url))[0] + '.pt')
-    if os.path.isfile(vectors_file_path):
-        return(torch.load(vectors_file_path))
-
-    downloaded_file_path = download_from_url(url, root=root)
+    cached_vectors_file_path = os.path.join(root, os.path.splitext(os.path.basename(url))[0] + '.pt')
+    if os.path.isfile(cached_vectors_file_path):
+        return(torch.load(cached_vectors_file_path))
+
+    # TODO(Nayef211): determine how to specify a relative file path
+    checksum = None
+    if validate_file:
+        with open("/private/home/nayef211/torchtext/torchtext/experimental/checksums_fast_text.json", 'r') as f:
```

Comment:
the folder directory?

Reply:
removed json files so this is not needed anymore